ci/pipeline/determine_tests_to_run.py

```diff
@@ -50,6 +50,14 @@ def is_pull_request():
     return event_type == "pull_request"
 
 
+def is_continuous_run():
+    # Periodically continuous run only on master branch
+    return (
+        os.environ.get("RAYCI_CONTINUOUS_RUN")
```

Comment:
we will set up a continuous/periodical run on postmerge that has this env variable set